### PR TITLE
docs: add article-byline readme

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/article-byline/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Article Byline
+
+An article byline is a piece of styledtext that consists of an author, or list
```
Comment (craigbilner) [inline packages/article-byline/README.md]: styled text

File: packages/article-byline/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+of authors, either with, or without author job titles. This data typically
```
Comment (craigbilner) [inline packages/article-byline/README.md]: can also include a location

Comment (colincclark) [reply packages/article-byline/README.md]: added

File: packages/article-byline/README.md (hunk 1 above)
Comment (colincclark) [reply packages/article-byline/README.md]: done

**Pull request merged**